corehq/apps/export/static/export/js/download_export.ng.js

```diff
@@ -151,7 +151,8 @@
                         + getFilterName($scope.exportList[0].export_type) + "]")
                         .val().split(','),
                     function (item) {
-                        if (item.substring(0,3) === "t__") {
+                        var prefix = "t__";
+                        if (item.substring(prefix.length) === prefix) {
                             return userTypes[item.substring(3)];
```

Comment:
```suggestion
                            return userTypes[item.substring(prefix.length)];
```

Reply:
Thanks, and glad I got to use the sweet new github feature

Reply:
apologies for mucking up the spacing

Reply:
no prob! https://github.com/dimagi/commcare-hq/pull/22140/commits/ce0ee9c2c0e0322fb5273122f54915acef270c3b